src/frontend/Komponenter/Personoversikt/LagBehandlingModal.tsx

```diff
@@ -108,10 +110,11 @@ const LagBehandlingModal: React.FunctionComponent<IProps> = ({
                 }}
             >
                 <option value="">Velg</option>
-                <option value={Behandlingstype.REVURDERING}>Revurdering</option>
+                {kanStarteRevurdering && (
```

Comment:
Burde den forsvinne, være "disabled" eller kunne velges, men at saksbehandler får en fornuftig feilmelding her?

Kan være litt pussig at den ikke finnes i lista - risikerer vi at de velger tilbakekreving for "å prøve" seg frem?

Reply:
☝️ bra poeng. Jeg stemmer på disabled?